app/api/orders.py

```diff
@@ -68,12 +68,9 @@ def is_payment_valid(order, mode):
         return (order.paid_via == 'paypal') and order.transaction_id
 
 
-def check_billing_info(data):
-    if data.get('amount') and data.get('amount') > 0 and not data.get('is_billing_enabled'):
-        raise UnprocessableEntity({'pointer': '/data/attributes/is_billing_enabled'},
-                                  "Billing information is mandatory for paid orders")
-    if data.get('is_billing_enabled') and not (data.get('company') and data.get('address') and data.get('city') and
-                                               data.get('zipcode') and data.get('country')):
+def check_billing_info(data, order):
+    if order.event.is_billing_info_mandatory and data.get('amount') and data.get('amount') > 0 and not (data.get('company')
+                            and data.get('address') and data.get('city') and data.get('zipcode') and data.get('country')):
```

Comment:
Move the billing info checks in a separate function

Reply:
Cool, Will change it